client/js/drawModule.js

```diff
@@ -0,0 +1,1073 @@
+'use strict';
+/*
+ * drawModule.js
+ * Copyright(c) 2016 Universidad de Las Palmas de Gran Canaria
+ * Authors:
+ *   Jaisiel Santana <jaisiel@gmail.com>
+ *   Alejandro Sánchez <alemagox@gmail.com>
+ *   Pablo Fernández <pablo.fernandez@ulpgc.es>
+ * MIT Licensed
+
+  This module contains the procedures to render
+  the restaurants and their information
+
+*/
+
+/*exported drawModule */
+var map; //map instance
+var connectionsAPI;
+var utils;
+var drawModule = (function() {
+
+  var maxRating = 5;
+  var alreadyPartySizeInit = false;
+  var minTime = {
+    hours: 12,
+    minutes: 30
+  };
+  var maxTime = {
+    hours: 22,
+    minutes: 30
+  };
+
+  //actions do nothing by default
+  var viewReservationAction = function() {};
```

Comment:
instead of repeating this son many times, better

var NOP = function() {};

and then var x = NOP, y= etc